### PR TITLE
Officers

### PR DESCRIPTION
**Pull request by jyxzhang**
Ported over the officers page (https://www.ocf.berkeley.edu/docs/about/officers/) from the old website to the new. I used a card based layout that I've seen several other clubs/organizations use. Hopefully the officers are okay with sharing their faces.

The Previous officers section is not implemented yet. I am not familiar with the old website, and when I looked at the old website, I couldn't figure out where the previous officers were stored. 

Currently, the officers are hardcoded in. It would probably be a good idea to store them in some other sort of file.
Let me know if there is anything else that should be done, or if you have any suggestions!

### CODE REVIEW

**Comment by ja5087** [comment]
Screenshots of how it currently looks for others reviewing: 

image
image

File: src/pages/About/Officers.vue [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+        id: 2,
+        name: "Saurabh Narain ",
+        handle: "snarain",
+        role: ["General Manager"],
+        face: "https://bulma.io/images/placeholders/1280x960.png",
+        icon: "https://bulma.io/images/placeholders/96x96.png"
+      },
+      {
+        id: 3,
+        name: "Nikhil Jha  ",
```
Comment (ja5087) [inline src/pages/About/Officers.vue:141]: whitespace

**Review by ja5087 — APPROVED** [review]
Happy to approve the general layout. In my opinion the normal text doesn't need to be bolded. We'll need to work on integrating old officers (could be a schema like dkessler mentioned), and pulling images possibly from Gravatar, but this is a great start!

File: src/pages/About/Officers.vue (hunk 1 above)
Comment (ja5087) [reply src/pages/About/Officers.vue:141]: Additionally this is data that can be pulled from LDAP, but understandably without ocfapi that might be difficult

**Comment by bentref** [comment]
I like your overall vision and layout, great job! A few suggestions: 

- I agree with ja5087 that the normal text should not be bolded.
- There's a lot of big text at the very top of the page, so the first cards are pushed pretty far down. I think it would look better if the Faculty Adviser and BoD parts were downsized or even moved to a different part of the page.
- There are a few possible options for tweaking the really wide text for the general and site managers, since it looks a bit awkward for it to be so much wider than the cards.
    - You could make the text the same width as the cards and center it above them
    - You could put the general and site managers sections *next to* each other, with some sort of vertical line or delineation between the two. 

Bottom line is, this is a great starting point for this page and thanks for your work on it! If you want to make some changes and push again we can merge. Of course there's no problem with approving the PR as-is as a work in progress, so it's up to you whether you want to merge this now or make some changes first.

File: src/pages/About/Officers.vue [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+          <div class="column">
+            <h3 class="title is-3 has-text-centered">Communications Heads</h3>
+            <OfficerCardSet
+              role-wanted="Communications Head"
+              :cards="cards"
+              :double-set="true"
+            />
+          </div>
+        </div>
+        <h3 class="title is-3 has-text-centered">DeCal Heads</h3>
```
Comment (bentref) [inline src/pages/About/Officers.vue:95]: Even though there is only one column of cards here, it might make sense to wrap this part in with the columns as well, just to be consistent so that these are all wrapped in column divs and on the same level in the structure.

**Comment by jyxzhang** [comment]
OK, I moved the Faculty Adviser and board of directors to the bottom (Hopefully Dr. Harvey won't mind) and unbolded the description text . I actually kind of liked the wide text descriptions under GM and SM because it makes it breaks up the big empty spaces on the sides; I would like other people's opinions on that.
